examples_tests/13.BRDF_Explorer/main.cpp

```diff
@@ -37,7 +36,7 @@ int main()
     video::IVideoDriver* driver = device->getVideoDriver();
     scene::ISceneManager* smgr = device->getSceneManager();
 
-    auto cegui = ext::cegui::createGUIManager(driver);
+    auto* brdf = new BRDF(driver);
```

Comment:
what is the `BRDF` object going to be responsible for?

Reply:
refactored